easybuild/easyconfigs/s/SignalP/SignalP-4.1f.eb

```diff
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'SignalP'
+version = '4.1f'
+
+homepage = 'http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp'
+description = """SignalP predicts the presence  and  location of signal peptide  cleavage sites
+in  amino acid  sequences  from  different organisms"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# no public download url
+# http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?signalp
+sources = ['signalp-%(version)s.Linux.tar.gz']
+checksums = ['413b0f6d8176dfa549ede35935154d418ff6198deb6d6ec8ac2e79ccb824644b']
+
+modextrapaths = {
+    'PATH': '',
+    'MANPATH': '',
+}
+
+modextravars = {
+    'SIGNALP': '%(installdir)s',
+}
+
+postinstallcmds = [
+    "sed -i -e 's|/var/tmp|/scratch|' %(installdir)s/signalp",
```

Comment:
I guess signalp is a bash script, if so it's probably better to try to make it use $TMPDIR instead of /scratch.
Not every site has it (even though we of course do :-)